pkg/storage/bucket/s3/config.go

```diff
@@ -269,6 +271,14 @@ func parseKMSEncryptionContext(data string) (map[string]string, error) {
 	return decoded, err
 }
 
+type TraceConfig struct {
+	Enable bool `yaml:"enable" category:"advanced"`
+}
+
+func (cfg *TraceConfig) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
+	f.BoolVar(&cfg.Enable, prefix+"enable", false, "If enabled, low-level S3 HTTP operation information is logged at the debug level.")
```

Comment:
(nit) I recall @pstibrany mentioned, that Mimir tries to stick to "enabled" for on/off settings.

```suggestion
	f.BoolVar(&cfg.Enable, prefix+"enabled", false, "If enabled, low-level S3 HTTP operation information is logged at the debug level.")
```

And the same for the fields in the config itself.

Reply:
Ah, thanks for that. I'll take a look at that this morning.